### PR TITLE
[11.0] [FIX] pos_order_mgmt: Load payment for old orders

### LINKED ISSUE #353: [11] [pos_order_mgmt] Payments not loaded in orders for old sessions
When you print a ticket generated in an old session, the payments do not appear.

### PR DESCRIPTION
**Pull request by carlos-domatix**
Fixed #353 .

### CODE REVIEW

File: pos_order_mgmt/README.rst [hunk 1]
```diff
@@ -104,6 +104,7 @@ Contributors
 
 * David Vidal <david.vidal@tecnativa.com>
 * Sylvain LE GAL (https://twitter.com/legalsylvain)
+* Carlos Martínez <carlos@domatix.com>
```
Comment (pedrobaeza) [inline pos_order_mgmt/README.rst]: You must do on readme/CONTRIBUTORS.rst, not here.

File: pos_order_mgmt/models/pos_order.py [hunk 2]
```diff
@@ -88,7 +88,7 @@ def _prepare_done_order_line_for_pos(self, order_line):
     def _prepare_done_order_payment_for_pos(self, payment_line):
         self.ensure_one()
         return {
-            'statement_id': payment_line.statement_id.id,
+            'journal_id': payment_line.journal_id.id,
```
Comment (pedrobaeza) [inline pos_order_mgmt/models/pos_order.py:91]: Why changing this?

Comment (chienandalu) [reply pos_order_mgmt/models/pos_order.py:91]: Well, makes sense since it's the right reference to match in the reprinted ticket.

Comment (pedrobaeza) [reply pos_order_mgmt/models/pos_order.py:91]: OK

File: pos_order_mgmt/README.rst (hunk 1 above)
Comment (carlos-domatix) [reply pos_order_mgmt/README.rst]: Done

**Review by chienandalu — APPROVED** [review]
Tested on runbot :+1: 

Thanks, @carlosDomatix !

**Review by pedrobaeza — APPROVED** [review]
Code review

**Pull request merged**